travis/install_grpc.sh

```diff
@@ -47,6 +55,52 @@ if [ -n "$grpc_dist" ]; then
   PIP=$grpc_dist/usr/local/bin/pip
   $PIP install --upgrade pip
   $PIP install --upgrade --ignore-installed virtualenv
+else
+  PIP=pip
+  $PIP install --upgrade pip
+  # System wide installations require an explicit upgrade of
+  # certain gRPC Python dependencies e.g. "six" on Debian Jessie.
+  $PIP install --upgrade --ignore-installed six
+fi
+
+# clone the repository, setup the submodules
+git clone https://github.com/grpc/grpc.git
+cd grpc
+git checkout v1.7.0
```

Comment:
I'm not sure if there is a relationship, but on go, we're on 1.7.1: https://github.com/youtube/vitess/blob/master/vendor/vendor.json#L980. May still be worth keeping them in sync, just in case.